### PR TITLE
Adding a new topic that includes info about upcoming breaking changes

### PR DESCRIPTION
**Pull request by randall-Mysten**
## Description 

Adds a new topic that describes the breaking changes coming to Sui with release .28

## Test Plan 

Manually

---
If your changes are not user-facing and not a breaking change, you can skip the following section. Otherwise, please indicate what changed, and then add to the Release Notes section as highlighted during the release process.

### Type of Change (Check all that apply)

- [ ] user-visible impact
- [ ] breaking change for a client SDKs
- [ ] breaking change for FNs (FN binary must upgrade)
- [ ] breaking change for validators or node operators (must upgrade binaries)
- [ ] breaking change for on-chain data layout
- [ ] necessitate either a data wipe or data migration

### CODE REVIEW

File: doc/src/doc-updates/sui-breaking-changes.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+**[Major breaking change]** - `SuiAddress` and `ObjectID` are now 32 bytes long instead of 20 bytes (in hex, the `len` increases from 40 to 64). If your software interacts with any `ObjectID` and `SuiAddress`, you must update it to use updated addresses and redeploy it. [PR 8542](https://github.com/MystenLabs/sui/pull/8542)
+
+**[Breaking change]** - Changes to Gas Budget to use SUI rather than gas units. This removes the concept of gas units from any user-related API operations. This does not change the format of `TransactionData` (u64). This is not a breaking change in the sense that the current format no longer works, but rather requires you to reconsider how you use gas budgets.
+
+**[Breaking change]** - Currently, transactions require a single coin to pay for gas. This sometimes results in users needing to make separate transactions (such as `PaySui`) to merge coins before completing a transaction, which can also increase the cost of the transaction.
+
```
Comment (srinath-mysten) [inline doc/src/doc-updates/sui-breaking-changes.md:14]: Remove linespace ?

File: doc/src/doc-updates/sui-breaking-changes.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
+---
+title: Sui Breaking Changes in Release .28
```
Comment (srinath-mysten) [inline doc/src/doc-updates/sui-breaking-changes.md:2]: the word "title" shows up in rtf view. Not sure if it's a problem.

File: doc/src/doc-updates/sui-breaking-changes.md (hunk 1 above)
Comment (randall-Mysten) [reply doc/src/doc-updates/sui-breaking-changes.md:14]: the whitespace makes it easier to read
image

File: doc/src/doc-updates/sui-breaking-changes.md (hunk 2 above)
Comment (randall-Mysten) [reply doc/src/doc-updates/sui-breaking-changes.md:2]: this is a standard markdown title
image

File: doc/src/doc-updates/sui-breaking-changes.md (hunk 1 above)
Comment (randall-Mysten) [reply doc/src/doc-updates/sui-breaking-changes.md:14]: could do bullets instead
image

Comment (ronny-mysten) [reply doc/src/doc-updates/sui-breaking-changes.md]: Even though the following section is "part" of the previous one on this change, I think the space is still preferred for readability. IMO, the bolding acts as bullets but don't think they're necessary. Wouldn't be mad if they're added, either, tho.

**Review by ronny-mysten — APPROVED** [review]

**Pull request merged**